File: WordPress/src/main/java/org/wordpress/android/ui/posts/JetpackSecuritySettingsActivity.java
package org.wordpress.android.ui.posts;

import android.os.Bundle;
import android.view.MenuItem;

import androidx.annotation.Nullable;
import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import org.wordpress.android.R;

public class JetpackSecuritySettingsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.fragment_jetpack_security_settings);

        setupToolbar();
    }

    private void setupToolbar() {
        setTitle(getResources().getText(R.string.jetpack_security_setting_title));

        Toolbar toolbar = findViewById(org.wordpress.mobile.ReactNativeGutenbergBridge.R.id.toolbar);
        if (toolbar != null) {
            setSupportActionBar(toolbar);

            ActionBar actionBar = getSupportActionBar();
            if (actionBar != null) {
                actionBar.setDisplayShowTitleEnabled(true);
                actionBar.setDisplayHomeAsUpEnabled(true);
                actionBar.setHomeAsUpIndicator(R.drawable.ic_close_24px);
                actionBar.setSubtitle("");
            }
        }
    }

    @Override
    public boolean onOptionsItemSelected(final MenuItem item) {
        int itemID = item.getItemId();

        if (itemID == android.R.id.home) {
            finish();
        }

        return super.onOptionsItemSelected(item);
    }
}

File: WordPress/src/main/java/org/wordpress/android/ui/posts/JetpackSecuritySettingsActivity.kt
package org.wordpress.android.ui.posts

import android.os.Bundle
import android.view.MenuItem
import androidx.activity.addCallback
import androidx.appcompat.app.AppCompatActivity
import org.wordpress.android.R.string
import org.wordpress.android.databinding.FragmentJetpackSecuritySettingsBinding

class JetpackSecuritySettingsActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        with(FragmentJetpackSecuritySettingsBinding.inflate(layoutInflater)) {
            setContentView(root)
            setupToolbar()
        }

        onBackPressedDispatcher.addCallback(this) {
            setResult(RESULT_OK, null)
            finish()
        }
    }

    private fun FragmentJetpackSecuritySettingsBinding.setupToolbar() {
        title = resources.getText(string.jetpack_security_setting_title)
        setSupportActionBar(toolbar)
        val actionBar = supportActionBar
        if (actionBar != null) {
            actionBar.setDisplayShowTitleEnabled(true)
            actionBar.setDisplayHomeAsUpEnabled(true)
        }
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        val itemID = item.itemId
        if (itemID == android.R.id.home) {
            setResult(RESULT_OK, null)
            finish()
            return true
        }
        return super.onOptionsItemSelected(item)
    }

    companion object {
        const val JETPACK_SECURITY_SETTINGS_REQUEST_CODE = 101
    }
}
